From dcbd307a8ae575d6ef50bb5b8708e621413fff1f Mon Sep 17 00:00:00 2001
From: Kentaro Fukuchi <kentaro@fukuchi.org>
Date: Wed, 15 May 2013 23:40:40 +0900
Subject: [PATCH 02/19] Some typing erros have been fixed.

---
 sl.1    | 14 +++++++-------
 sl.1.ja | 14 +++++++-------
 sl.c    |  2 +-
 sl.h    |  2 +-
 4 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/sl.1 b/sl.1
index 33337a6..c85d7f1 100644
--- a/sl.1
+++ b/sl.1
@@ -5,27 +5,27 @@
 .\"
 .TH SL 1 "May 5, 2013"
 .SH NAME
-sl ¥- cure your bad habit of mistyping
+sl \- cure your bad habit of mistyping
 .SH SYNOPSIS
 .B sl
 [
-.B ¥-alFc
+.B \-alFc
 ]
 .SH DESCRIPTION
 .B sl
 is a highly advanced animation program for curing your bad habit of mistyping.
 .PP
 .TP
-.B ¥-a
+.B \-a
 An accident is occurring. People cry for help.
 .TP
-.B ¥-l
+.B \-l
 Little version
 .TP
-.B ¥-F
+.B \-F
 It flies like the galaxy express 999.
 .TP
-.B ¥-c
+.B \-c
 C51 appears instead of D51.
 .PP
 .SH SEE ALSO
@@ -33,4 +33,4 @@ C51 appears instead of D51.
 .SH BUGS
 It sometimes list directory contents.
 .SH AUTHOR
-Toyoda Masashi（mtoyoda@acm.org)
+Toyoda Masashi (mtoyoda@acm.org)
diff --git a/sl.1.ja b/sl.1.ja
index 9396ebe..8a1274e 100644
--- a/sl.1.ja
+++ b/sl.1.ja
@@ -5,11 +5,11 @@
 .\"
 .TH SL 1 "May 5, 2013"
 .SH 名称
-sl ¥- キータイプを矯正します。
+sl \- キータイプを矯正します。
 .SH 形式
 .B sl
 [
-.B ¥-alF
+.B \-alF
 ]
 .SH 解説
 .B sl
@@ -17,16 +17,16 @@ sl ¥- キータイプを矯正します。
 .PP
 次のようなオプションがあります。
 .TP
-.B ¥-a
+.B \-a
 なにかアクシデントがあったようですね。助けを求める人々が悲痛です。
 .TP
-.B ¥-l
+.B \-l
 小さくなります。
 .TP
-.B ¥-F
+.B \-F
 飛びます。
 .TP
-.B ¥-c
+.B \-c
 C51になります。
 .PP
 .SH 関連事項
@@ -34,4 +34,4 @@ C51になります。
 .SH バグ
 カレントディレクトリの内容が表示されることがあります。
 .SH 著者
-豊田 正史（mtoyoda@acm.org)
+豊田 正史 (mtoyoda@acm.org)
diff --git a/sl.c b/sl.c
index c4b7535..4cd96c2 100644
--- a/sl.c
+++ b/sl.c
@@ -6,7 +6,7 @@
  *	Last Modified: 2013/ 5/ 5
  *========================================
  */
-/* sl version 5.00 : add -c option
+/* sl version 5.00 : add -c option                                           */
 /*                                              by Toyoda Masashi 2013/ 5/ 5 */
 /* sl version 4.00 : add C51, usleep(40000)                                  */
 /*                                              by Toyoda Masashi 2002/12/31 */
diff --git a/sl.h b/sl.h
index e14206e..903962b 100644
--- a/sl.h
+++ b/sl.h
@@ -9,7 +9,7 @@
 
 #define D51HIGHT	10
 #define D51FUNNEL	 7
-#define D51LENGTH       83
+#define D51LENGTH	83
 #define D51PATTERNS	 6
 
 
-- 
2.34.1

